webapp/src/components/cardDetail/comment.tsx

```diff
@@ -65,10 +82,9 @@ const Comment: FC<Props> = (props: Props) => {
                     </MenuWrapper>
                 )}
             </div>
-            <div
-                className='comment-text'
-                dangerouslySetInnerHTML={{__html: html}}
-            />
+            <Provider store={(window as any).store}>
```

Comment:
The Provider is necessary in this context because the function `messageHtmlToComponent` depends on certain state and actions that are part of the mmStore which is exposed on window object.